app/javascript/stylesheets/button.css

```diff
@@ -28,6 +28,12 @@
   color: white;
 }
 
+.button.white {
```

Comment:
Added new shared styles for the white button with the gray outline. I still want to add hover styling, but that's covered by #67.